packages/http-server/test/integration/http-server.integration.ts

```diff
@@ -184,18 +195,51 @@ describe('HttpServer (integration)', () => {
 
   it('converts host from [::] to [::1] in url', async () => {
     // Safari on MacOS does not support http://[::]:3000/
-    server = new HttpServer(dummyRequestHandler, {host: '::'});
+    server = givenHttpServer(dummyRequestHandler, {host: '::'});
     await server.start();
     expect(server.url).to.equal(`http://[::1]:${server.port}`);
   });
 
   it('converts host from 0.0.0.0 to 127.0.0.1 in url', async () => {
     // Windows does not support http://0.0.0.0:3000/
-    server = new HttpServer(dummyRequestHandler, {host: '0.0.0.0'});
+    server = givenHttpServer(dummyRequestHandler, {host: '0.0.0.0'});
     await server.start();
     expect(server.url).to.equal(`http://127.0.0.1:${server.port}`);
   });
 
+  it('supports HTTP/2 protocol with key and certificate files', async () => {
+    // For some reason, spdy fails with Node 11
+    if (+process.versions.node.split('.')[0] > 10) return;
```

Comment:
This early return will mark the test as passed, leading to false sense of security.

Please mark the test as skipped on Node.js 11.x. If we don't have a helper like "it.skipIf", then please consider creating one in `@loopback/testlab`.

```ts
it.skipIf(
  // For some reason, spdy fails with Node 11
  +process.versions.node.split('.')[0] > 10),
  'supports HTTP/2 protocol with key and certificate files',
  async () => {
    // ...
  }
);
```